harness/determined/pytorch/dsat/_defaults.py

```diff
@@ -62,7 +62,7 @@
 
 DEFAULT_SEARCH_RUNNER_CONFIG = {
     "searcher": {"name": "single", "max_length": 0},
-    "max_restarts": 5,
+    "max_restarts": 0,
```

Comment:
Note: I'm doing this so that it doesn't restart the orchestrator, realize the client experiment is finished (because it killed it last time) and then exiting "successfully". If it has only one chance to run, it will error out as (in my opinion) expected

Reply:
Perhaps not, but that's what I'm seeing in practice. I don't mind setting it back to 5, it's just that if an exception is raised on the orchestrator that way, it will end up shutting down the client experiment as having failed (which makes sense), but then shutting itself down as a success (which may or may not be what we want from a UX stand point)